charts/alertmanager/Chart.yaml

```diff
@@ -6,7 +6,7 @@ icon: https://raw.githubusercontent.com/prometheus/prometheus.github.io/master/a
 sources:
   - https://github.com/prometheus/alertmanager
 type: application
-version: 0.8.0
+version: 0.8.1
```

Comment:
New feature should raise minor version

Reply:
@monotek What should I put instead of 0.8.1 ?

Reply:
0.9.0